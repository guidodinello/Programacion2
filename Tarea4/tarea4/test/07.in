# Implementar binarioBalanceado
binarioBalanceado 1 50
imprimirBinario
binarioBalanceado 6 10 20 30 40 50 60
imprimirBinario
binarioBalanceado 7 10 20 30 40 50 60 70
imprimirBinario
binarioBalanceado 8 10 20 30 40 50 60 70 80
imprimirBinario
Fin
