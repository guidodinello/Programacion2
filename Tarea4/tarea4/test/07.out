1>#  Implementar binarioBalanceado.
2>
3>
(50,0.00)
4>
5>
--(60,0.00)
-(50,0.00)
--(40,0.00)
(30,0.00)
--(20,0.00)
-(10,0.00)
6>
7>
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
8>
9>
---(80,0.00)
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
10>Fin.
